### PR TITLE
Add loader before iFrame display

### PR DESCRIPTION
**Pull request by benmalcom**
- Add loader before iFrame content is fully loaded
- Add close button to drawer on mobile

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
   "name": "near-fastauth-wallet",
-  "version": "0.0.14",
+  "version": "0.0.21",
```
Comment (hcho112) [inline package.json]: Just wondering, instead of increment the version, is there a reason why it is 21?

**Review by hcho112 — APPROVED** [review]
LGTM, good improvements

File: package.json (hunk 1 above)
Comment (benmalcom) [reply package.json]: Ah, that was for testing with verdaccio, let me return it to the previous version.
Good catch.

Comment (benmalcom) [reply package.json]: Fixed

**Review by hcho112 — APPROVED** [review]

**Pull request merged**